tests/manage/test_expand_cluster_by_adding_backend_vol_rbd.py

```diff
@@ -0,0 +1,378 @@
+import os
+import pytest
+import logging
+from time import sleep
+import json
+
+from ocs_ci.ocs import constants, defaults, cluster
+from ocs_ci.utility.templating import load_yaml_to_dict
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.utility.aws import AWS
+from tests import helpers
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup(self):
+    """
+    This function just performs the following pre-requisite steps of this test.
+    1) Create secret
+    2) Create storageclass
+    3) Create a namespace specific to this test
+    4) Check the status of all the ocs pods - should be either in Running or Completed state
+    5) Check overall health of the ceph cluster
+    6) Create PVCs
+    7) Create Pods
+    :
+    :
+    """
+
+    # Create the secret
+    self.secret = helpers.create_secret(constants.CEPHBLOCKPOOL)
+
+    # Create storage class
+    self.sc = helpers.create_storage_class(constants.CEPHBLOCKPOOL, 'rbd', self.secret.name)
+
+    # TBD: create a namespace dedicated for app pods
+    # It's a good practice to run the app-pods in a project/namespace other than the openshift-storage namespace.
+
+    # Check status of all the ocs pods in openshift-storage namespace.
+    # They should be either in Running or Completed state
+    assert check_ocs_pods_status()
+
+    # Determine the health of the ceph cluster. If unhealthy do not proceed with testing.
+    # Note: Unhealthy = HEALTH_ERR, HEALTH_WARN
+    assert (cluster.CephCluster()).cluster_health_check()
+
+    # Create the rbd based PVCs so that app pods can run on them
+    number_of_pvcs_to_create = 2  # TBD: change this to parameterized format later. Accept it from cmdline
```

Comment:
regarding the comment - no from command line as we are not going to execute it like a script